test/test_filter_process_ucs_syslog.rb

```diff
@@ -211,6 +217,25 @@ def test_filter_bad_login
         end
         filtered_records = filter(records, BAD_LOGIN_CONFIG)
         assert_equal "", filtered_records[0]['machineId']
-        assert_equal "Unable to login to UCS to get service profile", filtered_records[0]['error']
+        assert_equal "Error getting service profile: Unable to login to UCS", filtered_records[0]['error']
+    end
+
+    def test_filter_bad_etcd_response
+        records = [
+            { 
+                "message" => ": 2018 May  3 00:05:36 IST: %UCSM-6-EVENT: [E4195921][8743116][transition][ucs-HANATDIT][] [FSM:BEGIN]: Soft shutdown of server sys/chassis-4/blade-7(FSM:sam:dme:ComputePhysicalSoftShutdown)",
+                "SyslogSource" => "1.1.1.2"
+            }
+        ]
+        filtered_records = filter(records)
+        assert_equal records[0]['message'], filtered_records[0]['message']
+        assert_equal 'Cisco_UCS:SJC2:org-root/org-T100/ls-testServiceProfile', filtered_records[0]['machineId']
```

Comment:
`SJC2` isn't a real colo; since it's not verified anywhere in these tests, I would be more comfortable using an obviously-test string: `'fakeHWType:fakeColo:testServiceProfile'`

Reply:
done

Reply:
ack